flang/test/Lower/OpenMP/target.f90

```diff
@@ -134,6 +134,94 @@ subroutine omp_target_exit_device
    !$omp target exit data map(from: a) device(d)
 end subroutine omp_target_exit_device
 
+!===============================================================================
+! Target_Update `to` clause
```

Comment:
Add tests to `flang/test/Lower/OpenMP/FIR/target.f90` as well.

Reply:
Added a combined test instead of one test per clause like I did for HLFIR since this is already covered in `Lower/OpenMP/target.f90`. Let me know if you prefer separate tests per clause here as well.

Reply:
I don't mind having either separate or combined tests, but I'd prefer them to be consistent across both test files.

Reply:
Done.